stable/redis-ha/templates/redis-ha-healthchecks.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "redis-ha.fullname" . }}-probes
+  labels:
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    app: {{ template "redis-ha.fullname" . }}
+data:
+  check-quorum.sh: |
+    #!/bin/sh
+    set -eu
+    MASTER_GROUP="{{ .Values.redis.masterGroupName }}"
+    SENTINEL_PORT={{ .Values.sentinel.port }}
+    REDIS_PORT={{ .Values.redis.port }}
+    NUM_SLAVES=$(redis-cli -p "$SENTINEL_PORT" sentinel master {{ .Values.redis.masterGroupName }} | awk '/num-slaves/{getline; print}')
+    MIN_SLAVES=$(({{ .Values.sentinel.quorum }} / 2))
```

Comment:
I think this should be `{ .Values.redis.config.min-slaves-to-write }`, no?

Reply:
That is a good point, I didn't consider that. I'll switch it and add a comment that it is a mandatory field for those that use the `customConfig` that overrides all the `.Values.redis.config` options.